docs/configuring-korrel8r.adoc

```diff
@@ -0,0 +1,117 @@
+[id="configuring-korrel8r"]
+= Configuring Korrel8r
+include::attributes.adoc[]
+
+You can configure how Korrel8r processes and generates rules, groups, and store configurations by creating or modifying a configuration file. Configuration files can be either JSON or YAML.
+
+You can pass a new configuration file to Korrel8r by running the following command:
+
+[source,terminal]
+----
+$ korrel8r --config <path_to_config_file>
+----
+
+.Example command
+[source,terminal]
+----
+$ korrel8r --config my-config.yaml
+----
+
+.Example YAML configuration file
+[source,yaml]
+----
+rules:
+  - name: <rule_name> # <1>
+    start:
+      domain: <domain_name> # <2>
+      classes:
+        - <list_of_classes_or_groups> # <3>
+    goal:
+      domain: <domain_name> # <4>
+      classes:
+        - <list_of_classes_or_groups> # <5>
+    result:
+      query: <query_name> # <6>
+      constraint: <constraint_name> # <7>
+groups:
+  - name: <group_name> # <8>
+    domain: <domain_name> # <9>
+    classes:
+      - <list_of_classes> # <10>
+stores:
+  - name: <store_name> # <11>
+    type: <store_type> # <12>
+    url: <url> # <13>
+include:
+  - <path_to_config_file> # <14>
+----
+<1> Specify a short, descriptive name for the rule.
+<2> Specify the domain that the rule can be applied to.
```

Comment:
This is all good, just food for thought about describing "start" and "goal". 
- "start" and "goal" are both classes - i.e. types of data object (possibly in separate domains)
- a rule is really a template for a query that returns goal objects, which is populated by data from start object(s). In that the rule is "applied" to an object of the start class and "produces" a goal query (which in turn can produce objects of the goal class)
- rules form the "edges" in a directed multi-graph where classes are "vertices" or "nodes". In graph theory (start,goal) would be called (u,v) or (from,to) 
- if rules were functions in a programming language "start" would be the parameter type and "goal" would be the return type.

Not suggesting any changes to the text here, it is nice and clear - just so you have the background. This can get confusing quickly if you're not careful with your words.